apps/gdevelop/gdevelop.yml

```diff
@@ -0,0 +1,25 @@
+name: 'GDevelop'
+description: 'Create games without programming.'
+website: 'https://gdevelop-app.com/'
+repository: 'https://github.com/4ian/GDevelop'
+keywords:
+  - 'Game Editor'
+  - 'Open Source'
+  - 'Cross-Platform '
+  - 'Visual Programming'
+  - 'Online Editor'
+  - 'Piskel Sprite Editor'
+  - 'Google AdMob'
+license: MIT
+category: 'Graphics & Design'
+npmPackageName: 'gdmod'
+youtube_video_url: 'https://www.youtube.com/watch?v=5jdOR-NAiSA'
+screenshots:
+  - imageUrl: 'https://github.com/4ian/GDevelop/blob/master/newIDE/gd-ide-screenshot.png'
+    caption: 'GDevelop Editor Dark Mode'
+  - imageUrl: 'https://github.com/4ian/GDevelop/blob/master/newIDE/integrated-ide-screenshot.png'
+    caption: 'GDevelop Editor Light Mode'
+  - imageUrl: 'https://raw.githubusercontent.com/4ian/GDevelop/master/newIDE/electron-app/build/icon.ico'
+    caption: 'GDevelop Logo'
+  - imageUrl: 'https://github.com/4ian/GDevelop-website/blob/master/src/img/events.png'
+    caption: 'GDevelop Events Page'
```

Comment:
As it's showing an event, let's go for `GDevelop visual events`

Reply:
Alright